<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmManCustomer</class>
 <widget class="QMainWindow" name="frmManCustomer">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CUSTOMER MANAGER</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gbTag">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>571</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="title">
     <string>TAG DETAILS</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>41</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>* CUSTOMER CODE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCode">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>142</x>
       <y>43</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtFname">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>142</x>
       <y>66</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>64</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>* FIRST NAME</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtMname">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>142</x>
       <y>89</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>87</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>* MIDDLE NAME</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtLname">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>143</x>
       <y>112</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>110</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>* LAST NAME</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkActive">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>23</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Active</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>16</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>STATUS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>ADDRESS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtAddress">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>369</x>
       <y>39</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtContact1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>369</x>
       <y>92</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>292</x>
       <y>88</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>CONTACT </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblPkeyCus">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>117</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>206</y>
      <width>571</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Century Gothic&quot;;
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLineEdit" name="txtCusNo_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>24</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>19</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>* CUSTOMER CODE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSearch">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>24</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>SEARCH</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="gbCustList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>551</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string>Record(s) Found</string>
     </property>
     <widget class="QTableWidget" name="tblCustList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>23</y>
        <width>531</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>CODE</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>FIRST</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>MIDDLE</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>LAST</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ADDRESS</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CONTACT 1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>NO</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>STATUS</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../Downloads/icons/1426825-sales/png/017-clerk-1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>12</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 24pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>CUSTOMER MANAGER</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>204</x>
      <y>541</y>
      <width>92</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>DELETE</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Downloads/icons/957216-ui-interface/png/010-close.png</normaloff>../../../../Downloads/icons/957216-ui-interface/png/010-close.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>391</x>
      <y>541</y>
      <width>92</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>SAVE</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Downloads/icons/957216-ui-interface/png/062-diskette.png</normaloff>../../../../Downloads/icons/957216-ui-interface/png/062-diskette.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>487</x>
      <y>541</y>
      <width>92</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>CANCEL</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Downloads/icons/957216-ui-interface/png/004-disabled.png</normaloff>../../../../Downloads/icons/957216-ui-interface/png/004-disabled.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>540</y>
      <width>92</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Downloads/icons/957216-ui-interface/png/090-plus.png</normaloff>../../../../Downloads/icons/957216-ui-interface/png/090-plus.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEdit">
    <property name="geometry">
     <rect>
      <x>107</x>
      <y>540</y>
      <width>92</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>EDIT</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Downloads/icons/957216-ui-interface/png/092-notes.png</normaloff>../../../../Downloads/icons/957216-ui-interface/png/092-notes.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>tblCustList</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>frmManCustomer</receiver>
   <slot>fetchTblRowDataCustomer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>523</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>582</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAdd</sender>
   <signal>clicked()</signal>
   <receiver>frmManCustomer</receiver>
   <slot>btnAddCusClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>574</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnEdit</sender>
   <signal>clicked()</signal>
   <receiver>frmManCustomer</receiver>
   <slot>btnEditCusClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>562</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>578</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDelete</sender>
   <signal>clicked()</signal>
   <receiver>frmManCustomer</receiver>
   <slot>btnDeleteCusClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>558</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>554</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSave</sender>
   <signal>clicked()</signal>
   <receiver>frmManCustomer</receiver>
   <slot>btnSaveCusClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>551</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>576</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>frmManCustomer</receiver>
   <slot>btnCancelCusClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>524</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>572</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btnAddCusClicked()</slot>
  <slot>btnEditCusClicked()</slot>
  <slot>btnDeleteCusClicked()</slot>
  <slot>btnSaveCusClicked()</slot>
  <slot>btnCancelCusClicked()</slot>
  <slot>fetchTblRowDataCustomer()</slot>
 </slots>
</ui>
